PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### :pushpin: References
+* **Issue:** _your issue goes here_
+* **Related pull-requests:** _list of related pull-requests (comma-separated): #1, #2_
+
+### :tophat: What is the goal?
+
+_Provide a description of the overall goal (you can usually copy the one from the issue)_
+
+### How is it being implemented?
+
+_Provide a description of the implementation_
+
+### How can it be tested?
```

Comment:
In case you want to add it, here is the emoji `:boom:`

Reply:
I've used a 🤖

Reply:
make sense!! 🤖💥